packages/web-components/src/components/va-memorable-date/va-memorable-date.tsx

```diff
@@ -192,6 +192,14 @@ export class VaMemorableDate {
   })
   componentLibraryAnalytics: EventEmitter;
 
+  componentDidLoad() {
+    // We are setting the error on each va-text-input for screen readers, but do not want to show it visually. 
+    const textInputs = this.el.shadowRoot.querySelectorAll('va-text-input');
+    textInputs.forEach((input) => {
+      input.shadowRoot.querySelector('#input-error-message').classList.add('sr-only');
+    });
```

Comment:
I'm ok with this approach. I think trying to add a conditional `sr-only` directly in the child components would make things more complicated than they need to be.

I just want to double check that we don't need to do this also with va-select when the month dropdown is used?

Reply:
I have added this solution to the month drop down, it also seems that validation was failing on the month drop down. I have added a quick fix for that so that the value of "0" will not pass validation.